binaryIoTypeC/proto/binaryIoTypeC/v1alpha1/binaryIoTypeC.proto

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright © 2022 Ci4Rail GmbH <engineering@ci4rail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// IEC IEC61131-2 compliant binary IO
+
+syntax = "proto3";
+
+package binaryIoTypeC;
+option go_package = "binaryIoTypeC/v1alpha1";
+
+// ============= Configuration =================
+enum ChannelMode {
+    // input channel
+    BINARYIOTYPEC_INPUT_TYPE_1_3 = 0;
+    // output channel, simple push pull
+    BINARYIOTYPEC_OUTPUT_PUSH_PULL = 1;
+    // further modes may be added later
+}
+
+message ChannelConfig {
+    ChannelMode mode = 1;
+    // If channel is configured for output, initialValue defines the value after initialization
+    // and after the watchdog has fired
+    bool initialValue = 2;
+}
+
+// ConfigurationSet to pass to Functionblock.Configuration.functionSpecificConfigurationSet hook
+message ConfigurationSet {
+    // per channel configuration (one per channel/pin)
```

Comment:
Add description which configuration belongs to which channel. E.g. the index is the channel number

Reply:
Ok. For each channel, a configuration must be passed. The index is the channel number.